src/Util/Casacore.cc

```diff
@@ -234,3 +234,46 @@ void NormalizeUnit(casacore::String& unit) {
         // not caught by check()
     }
 }
+
+bool ParseHistoryBeamHeader(std::string& header, std::string& bmaj, std::string& bmin, std::string& bpa) {
+    // Parse AIPS beam header using regex_match.
+    // Returns false if regex failed, else true with beam value-unit strings.
+    std::regex r;
+    std::cmatch results;
+    bool matched(false);
+
+    if (header.find("Beam") != std::string::npos) {
+        // Example:
+        // HISTORY RESTOR Beam =  2.000E+00 x  1.800E+00 arcsec, pa =  8.000E+01 degrees
+        r = ".*Beam\\s*=\\s*([\\d\\.Ee+\\-]+)\\s*x\\s*([\\d\\.Ee+\\-]+)\\s*([A-Za-z]*)\\s*\\,*\\s*pa\\s*=\\s*([\\d\\.Ee+\\-]+)\\s*([A-Za-z]"
```

Comment:
This is just a legibility issue, but we could use raw string literals for these regexes to eliminate the double escapes. It also shouldn't be necessary to escape `.` inside `[]`, or `-` if it's at the end of the `[]`, or to escape `,` (I think it's only a special character inside `{}`).

Reply:
There's a stray `\\.` in the second `[]` in the first regex.

I have also verified that you can write these as `R"/(.*Beam\s*=\s*([\d.Ee+-]+)\s*x\s*([\d.Ee+-]+)\s*([A-Za-z]*)\s*,*\s*pa\s*=\s*([\d.Ee+-]+)\s*([A-Za-z]*).*)/"` and `R"/(.*BMAJ\s*=\s*([\d.Ee+-]+)\s*BMIN\s*=\s*([\d.Ee+-]+)\s*BPA\s*=\s*([\d.Ee+-]+).*)/"` -- they evaluate to the same strings. I picked `/` as the delimiter (which is required because there are parentheses inside the string).